src/ir_Hitachi.cpp

```diff
@@ -859,30 +967,19 @@ void IRsend::sendHitachiAc424(const uint8_t data[], const uint16_t nbytes,
 #endif  // SEND_HITACHI_AC424
 
 #if DECODE_HITACHI_AC424
-// Decode the supplied Hitachi 424 bit A/C message.
-//
-// Note: This protocol is almost exactly the same as HitachiAC2 except this
-//       variant has a leader section as well, and subtle timing differences.
-//       It is also in LSBF order (per byte), rather than MSBF order.
-//
-// Args:
-//   results: Ptr to the data to decode and where to store the decode result.
-//   offset:  The starting index to use when attempting to decode the raw data.
-//            Typically/Defaults to kStartOffset.
-//   nbits:   The number of data bits to expect. Typically kHitachiAc424Bits.
-//   strict:  Flag indicating if we should perform strict matching.
-// Returns:
-//   boolean: True if it can decode it, false if it can't.
-//
-// Status: STABLE / Reported as working.
-//
-// Supported devices:
-//  Hitachi Shirokumakun / AC Model: RAS-AJ25H / AC Remote Model: RAR-8P2
-//  Manual (Japanese):
-//    https://kadenfan.hitachi.co.jp/support/raj/item/docs/ras_aj22h_a_tori.pdf
-//
-// Ref:
-//   https://github.com/crankyoldgit/IRremoteESP8266/issues/973
+/// Decode the supplied Hitachi 53-byte/424-bit A/C message.
+/// Status: STABLE / Reported as working.
+/// @param[in,out] results Ptr to the data to decode & where to store the result
+/// @param[in] offset The starting index to use when attempting to decode the
+///   raw data. Typically/Defaults to kStartOffset.
+/// @param[in] nbits The number of data bits to expect.
+/// @param[in] strict Flag indicating if we should perform strict matching.
+/// @return True if it can decode it, false if it can't.
+/// @note This protocol is almost exactly the same as HitachiAC2 except this
+///   variant has a leader section as well, and subtle timing differences.
+///   It is also in LSBF order (per byte), rather than MSBF order.
+/// @see https://github.com/crankyoldgit/IRremoteESP8266/issues/973
+/// @see https://kadenfan.hitachi.co.jp/support/raj/item/docs/ras_aj22h_a_tori.pdf
```

Comment:
any chance we can keep information about "Manual (Japanese)" ?

Reply:
Done.